deploy.sh

```diff
@@ -45,9 +45,7 @@ build_push_images() {
 	# "docker manifest" requires experimental feature enabled
 	export DOCKER_CLI_EXPERIMENTAL=enabled
 
-	# get baseimg (ceph/ceph:tag)
-	dockerfile="deploy/cephcsi/image/Dockerfile"
-	baseimg=$(awk '/^FROM/ {print $NF}' "${dockerfile}")
+	baseimg="ceph/ceph:v15"
```

Comment:
Can you detect the version? Maybe (untested) with

    baseimg="$(awk -F= '/^ARG BASE_IMG=/{print $NF}' deploy/cephcsi/image/Dockerfile)"